files/build_templates/init_cfg.json.j2

```diff
@@ -183,5 +183,11 @@
             "admin_state": "enabled",
             "vrf": "default"
         }
+    },
+    "STORMOND_CONFIG": {
+        "INTERVALS": {
+            "daemon_polling_interval" : "",
```

Comment:
I don' think this can pass yang validation. Can you add a test using this sample in yang?
```
      "INTERVALS": {
            "daemon_polling_interval" : "",
            "fsstats_sync_interval"   : ""
        }
```

Reply:
Modified per our offline discussion